sky/skylet/providers/command_runner.py

```diff
@@ -72,6 +83,9 @@ def docker_start_cmds(
         env_flags,
         user_options_str,
         '--net=host',
+        # SkyPilot: Add --privileged and --device=/dev/fuse to enable fuse.
+        '--privileged',
```

Comment:
May want to consider using `--cap-add SYS_ADMIN --device /dev/fuse --security-opt apparmor:unconfined` instead, since `--privileged` may be a security concern and [breaks GPU isolation](https://github.com/NVIDIA/nvidia-docker/issues/764) :)